src/entities/SurveyTopic/decoder.test.ts

```diff
@@ -14,44 +12,41 @@ const SENTIMENT_SURVEY: TopicFeedback[] = [
 ]
 
 describe('decode', () => {
-  def('decodedSurvey', () => decodeSurvey(get.encodedSurvey))
+  const decodedSurvey = (encodedSurvey: Record<string, unknown>) => decodeSurvey(encodedSurvey)
 
   describe('and empty survey', () => {
-    def('encodedSurvey', () => '')
+    // TODO: encodedSurvey shouldn't be a string. Maybe { survey: [] }?
```

Comment:
I'm not sure about this.

The survey encodes to a string because we are saving a stringifyed version of the survey in snapshot's `reason` field. 

I don't remember the details, but there could be old proposals with an empty reason, and I think snapshot returns an empty or null value.

Reply:
Also, it's possible to send a custom or empty reason voting directly from snapshot, so we have no guarantee about what we are going to receive

Reply:
Then we should update decodeSurvey params to be `encodedSurvey?: string | Record<string, unknown>`.
I'd remove the TODO. We could review this another time.